scopes/react/react/webpack/webpack.config.base.ts

```diff
@@ -70,13 +71,13 @@ export default function (isEnvProduction = false): Configuration {
       extensions: moduleFileExtensions.map((ext) => `.${ext}`),
 
       alias: {
-        'react/jsx-dev-runtime': require.resolve('react/jsx-dev-runtime.js'),
-        'react/jsx-runtime': require.resolve('react/jsx-runtime.js'),
-        'react-dom/server': require.resolve('react-dom/server'),
+        [join('react', 'jsx-dev-runtime')]: require.resolve('react/jsx-dev-runtime.js'),
+        [join('react', 'jsx-runtime')]: require.resolve('react/jsx-runtime.js'),
+        [join('react-dom', 'server')]: require.resolve('react-dom/server'),
```

Comment:
is this really required? i'm pretty sure webpack should know to handle this

Reply:
we've seen a lot of issues with jsx-runtime not being loaded recently. Maybe it's just for windows users and webpack doesnt deal with this well?